internal/collector/telemetry/process_telemetry.go

```diff
@@ -128,14 +155,18 @@ func (pmv *ProcessMetricsViews) StopCollection() {
 
 func (pmv *ProcessMetricsViews) updateViews() {
 	ms := &runtime.MemStats{}
+	stats.Record(context.Background(), mUptime.M(time.Now().UnixNano()-pmv.startTimeUnixNano))
 	pmv.readMemStats(ms)
 	stats.Record(context.Background(), mRuntimeAllocMem.M(int64(ms.Alloc)))
 	stats.Record(context.Background(), mRuntimeTotalAllocMem.M(int64(ms.TotalAlloc)))
 	stats.Record(context.Background(), mRuntimeSysMem.M(int64(ms.Sys)))
 
 	if pmv.proc != nil {
-		if procStat, err := pmv.proc.Stat(); err == nil {
-			stats.Record(context.Background(), mCPUSeconds.M(int64(procStat.CPUTime())))
+		if times, err := pmv.proc.Times(); err == nil {
+			stats.Record(context.Background(), mCPUSeconds.M(int64(times.Total())))
```

Comment:
This should really be a double but haven't changed it to maintain backwards compatibility

Reply:
We should plan to fix this in a follow-up.